pinot-core/src/main/java/org/apache/pinot/core/transport/grpc/GrpcQueryServer.java

```diff
@@ -72,19 +77,38 @@ public class GrpcQueryServer extends PinotQueryServerGrpc.PinotQueryServerImplBa
   private final AccessControl _accessControl;
   private final ServerQueryLogger _queryLogger = ServerQueryLogger.getInstance();
 
+  // Filter to keep track of gRPC connections.
+  private class GrpcQueryTransportFilter extends ServerTransportFilter {
+    @Override
+    public Attributes transportReady(Attributes transportAttrs) {
+      LOGGER.info("gRPC transportReady: REMOTE_ADDR {}",
```

Comment:
Will this potentially flood the log? Since we already have the metrics, maybe no need to log it

Reply:
Proxy creates the transport when it gets the first query for a server, and does not close it, so no risk of flooding the log file.